src/basic_types.jl

```diff
@@ -76,20 +76,33 @@ struct BoundaryConditionDoNothing end
 # This version can be called by hyperbolic solvers on logically Cartesian meshes
 @inline function (::BoundaryConditionDoNothing)(
     u_inner, orientation_or_normal_direction, direction::Integer, x, t, surface_flux, equations)
+
   return flux(u_inner, orientation_or_normal_direction, equations)
+  # TODO: should this be switched to `surface_flux(u_inner, u_inner, orientation_or_normal_direction, equations)` for consistency?
```

Comment:
```suggestion
```
Or is this question still open?

Reply:
It's still open. I can go ahead and replace it for consistency and see if it breaks any tests?

Reply:
Nevermind, lets remove this comment and leave this part as is for now. If we switch to @andrewwinters5000 treatment of nonconservative boundary fluxes, this won't be an issue anyways.